### PR TITLE
V4/chart annotations

### PR DESCRIPTION
**Pull request by PatrickCleary**
## Motivation

Feedback from labs folks

## Changes

Adds baselines (highest monthly median scheduled service)

And blocks for days with shuttling/no data

Screen Shot 2023-05-09 at 10 42 27 AM

## Testing Instructions
Try out different date ranges

### CODE REVIEW

File: modules/service/ServiceGraph.tsx [hunk 1]
```diff
@@ -92,6 +97,11 @@ export const ServiceGraph: React.FC<ServiceGraphProps> = ({
             ),
             backgroundColor: pattern.draw('diagonal', '#FFFFFF', lineColor, 5),
           },
+          {
+            label: 'Baseline',
```
Comment (devinmatte) [inline modules/service/ServiceGraph.tsx:101]: We will need to describe in the UI what baseline means. We probably should be adding a section to most pages that describes the dataset and what everything means more clearly

Side note: We should add some kind of baseline to our other graphs too!
- Travel times showing MBTA Benchmark and Transitmatters Benchmark/Baseline!

**Comment by devinmatte** [comment]
Screenshot 2023-05-08 at 10 14 37 AM
Screenshot 2023-05-08 at 10 14 49 AM
This looks great, but also depressing

File: modules/service/ServiceGraph.tsx (hunk 1 above)
Comment (PatrickCleary) [reply modules/service/ServiceGraph.tsx:101]: Yea, completely agree. I think we probably want to add a blurb to every widget. Maybe 1 sentence directly on the widget and an info tooltip with more.

And yea, definitely want to add more baselines.

Comment (devinmatte) [reply modules/service/ServiceGraph.tsx:101]: Maybe an expander like we use in https://dashboard-v4-beta.labs.transitmatters.org/opensource/ under some widgets with a description? I feel like we should make it easier to read than a tooltip.
But anyway, this looks great so far

**Review by devinmatte — APPROVED** [review]

**Review by skaplan-dev — APPROVED** [review]

**Pull request merged**